README.md

```diff
@@ -51,6 +51,7 @@ Whether to remove the taint that denies pods from being deployed to the Kubernet
 
 Whether to enable the Kubernetes web dashboard UI (only accessible on the master itself, or proxied), and the file containing the web dashboard UI manifest.
 
+    kubernetes_pod_network_plugin: 'flannel'
```

Comment:
This should probably be a separate docs line and description (below the variable), and should mention the available options (as of this PR, `flannel` or `kube-router`.

Reply:
I have added a few sentences documenting the use of this parameter, as well as the currently supported options.